app/src/cases/view/casePrintDialog.html

```diff
@@ -0,0 +1,84 @@
+<md-dialog aria-label="">
+  <form name="uploadForm">
+    <md-toolbar layout="row">
+      <div class="md-toolbar-tools">
+        <h2>{{ 'CASE.PRINT_CASE' | translate }} "{{dlg.case["cm:title"].value}}"</h2>
+      </div>
+    </md-toolbar>
+    <md-dialog-content style="max-width:800px;max-height:810px;min-width:20em">
+    
+        <md-checkbox ng-model="dlg.caseDetails"  aria-label="checkbox">
```

Comment:
The aria-label attribute is used to define a string that labels the current element. This one should be named "Details"